.vib/prometheus/goss/prometheus.yaml

```diff
@@ -0,0 +1,4 @@
+command:
```

Comment:
Looking at the Dockerfile's `ENTRYPOINT & CMD`, I think we can easily check the prometheus server runs correctly by just runnning:

```
timeout --preserve-status 5 prometheus --config.file=/opt/bitnami/prometheus/conf/prometheus.yml
```

Reply:
Great suggestion! I will try to apply that to the other PRs when possible.